asset/src/main/java/com/example/asset/AnalyzeOrgPoliciesExample.java

```diff
@@ -25,35 +25,28 @@
 
 public class AnalyzeOrgPoliciesExample {
 
-  public static void main(String[] args) {
+  public static void main(String[] args) throws Exception {
     // TODO(developer): Replace the ORG_ID with your Google Cloud Organization ID
     String scope = "organizations/ORG_ID";
     // TODO(developer): Replace the CONSTRAINT_NAME with the name of the constraint
-    // you want to analyze. Find more details at:
-    // https://cloud.google.com/policy-intelligence/docs/analyze-organization-policies
+    // you want to analyze. Find more Organization Policy Constraints at:
+    // "http://cloud/resource-manager/docs/organization-policy/org-policy-constraints"
     String constraint = "constraints/CONSTRAINT_NAME";
     analyzeOrgPolicies(scope, constraint);
   }
 
   // Analyzes accessible Org policies that match a request.
-  public static void analyzeOrgPolicies(String scope, String constraint) {
+  public static void analyzeOrgPolicies(String scope, String constraint) throws Exception {
     AnalyzeOrgPoliciesRequest request =
         AnalyzeOrgPoliciesRequest.newBuilder().setScope(scope).setConstraint(constraint).build();
 
     // Initialize client that will be used to send requests. This client only needs to be created
     // once, and can be reused for multiple requests. After completing all of your requests, call
     // the "close" method on the client to safely clean up any remaining background resources.
-    try (AssetServiceClient client = AssetServiceClient.create()) {
-      AnalyzeOrgPoliciesPagedResponse response = client.analyzeOrgPolicies(request);
-      System.out.println(
-          "AnalyzeOrgPolicies completed successfully:\n" + response.getPage().getValues());
-    } catch (IOException e) {
-      e.printStackTrace();
-      System.out.println("Failed to create client:\n" + e.toString());
-    } catch (ApiException e) {
-      e.printStackTrace();
-      System.out.println("Error during AnalyzeOrgPolicies:\n" + e.toString());
-    }
+    AssetServiceClient client = AssetServiceClient.create();
```

Comment:
Please keep the try around the client initialization for all samples (https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md#client-initialization)

Reply:
Done. Thank you!